### PR TITLE
Changelog for 1.10.2

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.2](https://github.com/go-gitea/gitea/releases/tag/v1.10.2) - 2020-01-02
+* BUGFIXES
+  * [API] Allow only specific Colums to be updated on Issue (#9539) (#9580)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Please remove `[API]`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add ErrReactionAlreadyExist error (#9550) (#9564)
+  * Fix bug when migrate from API (#8631) (#9563)
+  * Use default avatar for ghost user (fix 500 error) (#9536) (#9537)
+  * Fix repository issues pagination bug when there are more than one label filter (#9512) (#9528)
+  * Fix deleted branch isn't removed when push the branch again (#9516) (#9524)
+  * Fix missing repository status when migrating repository via API (#9511)
+  * Trigger webhook when deleting a branch after merging a PR (#9510)
+  * [API] Fix paging on /repos/{owner}/{repo}/git/trees/{sha} endpoint (#9482)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Please remove `[API]`

File: CHANGELOG.md (hunk 1 above)
Comment (techknowlogick) [reply CHANGELOG.md]: `Colums` -> `Columns`

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.2](https://github.com/go-gitea/gitea/releases/tag/v1.10.2) - 2020-01-02
+* BUGFIXES
+  * [API] Allow only specific Colums to be updated on Issue (#9539) (#9580)
+  * Add ErrReactionAlreadyExist error (#9550) (#9564)
+  * Fix bug when migrate from API (#8631) (#9563)
+  * Use default avatar for ghost user (fix 500 error) (#9536) (#9537)
```
Comment (techknowlogick) [inline CHANGELOG.md]: please remove `(fix 500 error)`

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix repository issues pagination bug when there are more than one label filter (#9512) (#9528)
+  * Fix deleted branch isn't removed when push the branch again (#9516) (#9524)
```
Comment (techknowlogick) [inline CHANGELOG.md]: `isn't` -> `not`

**Review by techknowlogick — COMMENTED** [review]
few nits

File: CHANGELOG.md (hunk 2 above)
Comment (6543) [reply CHANGELOG.md]: @techknowlogick instead of [API] can i add  ... via API at the end?

Comment (techknowlogick) [reply CHANGELOG.md]: maybe add `API` in front of `endpoint`?

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.2](https://github.com/go-gitea/gitea/releases/tag/v1.10.2) - 2020-01-02
+* BUGFIXES
+  * Allow only specific Colums to be updated on Issue via (#9539) (#9580)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Allow only specific columns to be updated on Issue via API (#9539) (#9580)
```

**Comment by 6543** [comment]
@techknowlogick  ... API ... is now good?

**Comment by 6543** [comment]
PS: we should cancle useless drone tasks ...

File: CHANGELOG.md (hunk 5 above)
Comment (techknowlogick) [reply CHANGELOG.md]: @6543 You added the via at the end of this so in that case API should be added at end too

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.2](https://github.com/go-gitea/gitea/releases/tag/v1.10.2) - 2020-01-02
+* BUGFIXES
+  * Allow only specific Columns to be updated on Issue via (#9539) (#9580)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Allow only specific Columns to be updated on Issue via API (#9539) (#9580)
```

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
👍 thanks

**Review by lunny — APPROVED** [review]

**Pull request merged**